src/test/java/uk/gov/hmcts/reform/pip/data/management/service/ValidationServiceTest.java

```diff
@@ -0,0 +1,235 @@
+        headerMap.put(PublicationConfiguration.SENSITIVITY_HEADER, SENSITIVITY);
+        headerMap.put(PublicationConfiguration.LANGUAGE_HEADER, LANGUAGE);
+        headerMap.put(PublicationConfiguration.DISPLAY_FROM_HEADER, null);
+        headerMap.put(PublicationConfiguration.DISPLAY_TO_HEADER, DISPLAY_TO);
+        HeaderValidationException dateHeaderValidationException =
+            assertThrows(HeaderValidationException.class, () -> {
+                validationService.validateHeaders(headerMap);
+            });
+        assertEquals("x-display-from Field is required for artefact type LIST",
+                     dateHeaderValidationException.getMessage(), VALIDATION_EXPECTED_MESSAGE
+        );
+    }
+
+    @Test
+    void testCreationOfPublicationJudgementTypeAndEmptyDateFrom() {
+        Map<String, Object> headerMap = new HashMap<>();
+        headerMap.put(PublicationConfiguration.PROVENANCE_HEADER, PROVENANCE);
+        headerMap.put(PublicationConfiguration.TYPE_HEADER, ArtefactType.JUDGEMENT);
+        headerMap.put(PublicationConfiguration.SOURCE_ARTEFACT_ID_HEADER, SOURCE_ARTEFACT_ID);
+        headerMap.put(PublicationConfiguration.SENSITIVITY_HEADER, SENSITIVITY);
+        headerMap.put(PublicationConfiguration.LANGUAGE_HEADER, LANGUAGE);
+        headerMap.put(PublicationConfiguration.DISPLAY_FROM_HEADER, null);
+        headerMap.put(PublicationConfiguration.DISPLAY_TO_HEADER, DISPLAY_TO);
+        HeaderValidationException dateHeaderValidationException =
+            assertThrows(HeaderValidationException.class, () -> {
+                validationService.validateHeaders(headerMap);
+            });
+        assertEquals("x-display-from Field is required for artefact type JUDGEMENT",
+                     dateHeaderValidationException.getMessage(), VALIDATION_EXPECTED_MESSAGE
+        );
+    }
+
+    @Test
+    void testCreationOfPublicationOutcomeTypeAndEmptyDateFrom() {
+        Map<String, Object> headerMap = new HashMap<>();
+        headerMap.put(PublicationConfiguration.PROVENANCE_HEADER, PROVENANCE);
+        headerMap.put(PublicationConfiguration.TYPE_HEADER, ArtefactType.OUTCOME);
+        headerMap.put(PublicationConfiguration.SOURCE_ARTEFACT_ID_HEADER, SOURCE_ARTEFACT_ID);
+        headerMap.put(PublicationConfiguration.SENSITIVITY_HEADER, SENSITIVITY);
+        headerMap.put(PublicationConfiguration.LANGUAGE_HEADER, LANGUAGE);
+        headerMap.put(PublicationConfiguration.DISPLAY_FROM_HEADER, null);
+        headerMap.put(PublicationConfiguration.DISPLAY_TO_HEADER, DISPLAY_TO);
+        HeaderValidationException dateHeaderValidationException =
+            assertThrows(HeaderValidationException.class, () -> {
+                validationService.validateHeaders(headerMap);
+            });
+        assertEquals("x-display-from Field is required for artefact type OUTCOME",
+                     dateHeaderValidationException.getMessage(), VALIDATION_EXPECTED_MESSAGE
+        );
+    }
```

Comment:
youll need to make sure all enums are covered, itll be mostly copy paste but that way weve covered all scenarios with the enum and date provided/not provided combination